src/pugixml.cpp

```diff
@@ -10863,6 +10865,7 @@ PUGI__NS_BEGIN
 			if (_next)
 				_next->optimize(alloc);
 
+			// coverity[var_deref_model]
```

Comment:
Is this required given the change https://github.com/zeux/pugixml/commit/d9fadc74646171d72b9a539ecff7183de1228f00?

Reply:
Commit https://github.com/zeux/pugixml/commit/d9fadc74646171d72b9a539ecff7183de1228f00 addressed the 4. issue in line 10926

Reply:
Maybe @cecilios can have a look and test with Coverity again

Reply:
@c72578 Now quite bussy but I will check on this weekend.